run.sh

```diff
@@ -30,60 +29,45 @@ function build_cokapi_containers {
 function build_grading_containers {
   echo "building grading containers"
   echo "building base container"
-  [[ "$(docker images -q ingi/inginious-c-base 2> /dev/null)" == "" ]] && docker build -t ingi/inginious-c-base $JUDGE_HOME/INGInious/base-containers/base
+  [[ "$(docker images -q ingi/inginious-c-base 2> /dev/null)" == "" ]] && docker build -t ingi/inginious-c-base $JUDGE_TEMP/INGInious/base-containers/base
   echo "building default container"
-  [[ "$(docker images -q ingi/inginious-c-default 2> /dev/null)" == "" ]] && docker build -t ingi/inginious-c-default $JUDGE_HOME/INGInious/base-containers/default/
+  [[ "$(docker images -q ingi/inginious-c-default 2> /dev/null)" == "" ]] && docker build -t ingi/inginious-c-default $JUDGE_TEMP/INGInious/base-containers/default/
   echo "building multilang container"
-  [[ "$(docker images -q ingi/inginious-c-multilang 2> /dev/null)" == "" ]] && docker build -t ingi/inginious-c-multilang $JUDGE_HOME/INGInious-containers/grading/multilang
+  [[ "$(docker images -q ingi/inginious-c-multilang 2> /dev/null)" == "" ]] && docker build -t ingi/inginious-c-multilang $JUDGE_TEMP/INGInious-containers/grading/multilang
 }
 
-function move_config_file {
+function move_config_files {
   echo "setting up config file"
-  mv ./configuration.yaml $JUDGE_HOME
+  cp ./configuration.yaml /var/www/INGInious/
```

Comment:
Isn't it better to have this: `/var/www/INGInious/` as an ENV variable? you are using it in several lines and files.